.storybook/utils/styled-system-props.js

```diff
@@ -140,30 +140,329 @@ const generateStyledSystemProps = (
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.justifyItems || '-'
+      }
+    },
+    {
+      name: 'justifyContent',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.justifyContent || '-'
+      }
+    },
+    {
+      name: 'flexWrap',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.flexWrap || '-'
+      }
+    },
+    {
+      name: 'flexDirection',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.flexDirection || '-'
+      }
+    },
+    {
+      name: 'flex',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.flex || '-'
+      }
+    },
+    {
+      name: 'flexGrow',
+      type: { summary: 'number' },
+      description: 'Any number greater than 0.',
+      required: false,
+      defaultValue: {
+        summary: defaults.flexGrow || '-'
+      }
+    },
+    {
+      name: 'flexShrink',
+      type: { summary: 'number' },
+      description: 'Any number greater than 0.',
+      required: false,
+      defaultValue: {
+        summary: defaults.flexShrink || '-'
+      }
+    },
+    {
+      name: 'flexBasis',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.flexBasis || '-'
+      }
+    },
+    {
+      name: 'justifySelf',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.justifySelf || '-'
+      }
+    },
+    {
+      name: 'alignSelf',
+      type: { summary: 'string' },
+      description: 'Any valid CSS string',
+      required: false,
+      defaultValue: {
+        summary: defaults.alignSelf || '-'
+      }
+    },
+    {
+      name: 'order',
+      type: { summary: 'number' },
+      description: 'Any number greater than 0.',
+      required: false,
+      defaultValue: {
+        summary: defaults.order || '-'
+      }
+    }
+  ];
+};
+
 const StyledSystemProps = ({
   of,
   spacing,
-  spacingDefaults = {},
+  color,
+  layout,
+  flexBox,
+  defaults = {},
   noHeader
 }) => {
-  const rows = spacing ? generateStyledSystemProps(spacingDefaults) : null;
-
+  let sections = {};
+  if(spacing)
+  sections['Spacing'] = generateStyledSystemSpacingProps(defaults);
+  if(color)
+  sections['Color'] = generateStyledSystemColorProps(defaults);
+  if(layout)
+  sections['Layout'] = generateStyledSystemLayoutProps(defaults);
+  if(flexBox)
+  sections['FlexBox'] = generateStyledSystemFlexBoxProps(defaults);
```

Comment:
I think that in the new prettier settings all single quotes were changed to double quotes